src/test/java/io/managed/services/test/rhoas/KafkaRhoasAclTests.java

```diff
@@ -0,0 +1,367 @@
+        assertTrue(aclFiltered.isPresent());
+    }
+
+    @SneakyThrows
+    @Test(dependsOnMethods = "testCreateAclForUser", enabled = true)
+    public void testDeleteAclForUser() {
+        LOGGER.info("Create ACL for secondary user");
+        cli.deleteAcl(DIFF_USER, AclOperation.ALL, AclPermissionType.ALLOW);
+        var aclList = cli.listACLs(DIFF_USER);
+        Optional<AclBinding> aclFiltered = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.ALL, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        assertTrue(aclFiltered.isEmpty());
+    }
+
+    @SneakyThrows
+    @Test(priority = 2)
+    public void testCreateAclForServiceAccount() {
+        LOGGER.info("Create ACL for service account");
+        cli.createAcl(serviceAccount, AclOperation.ALL, AclPermissionType.ALLOW, TOPIC_NAME);
+        var aclList = cli.listACLs(serviceAccount);
+        Optional<AclBinding> aclFiltered = CLIUtils.searchAcl(aclList, serviceAccount, AclOperation.ALL, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        assertTrue(aclFiltered.isPresent());
+    }
+
+    @SneakyThrows
+    @Test(dependsOnMethods = "testCreateAclForServiceAccount", enabled = true)
+    public void testDeleteAclForServiceAccount() {
+        LOGGER.info("Delete ACL for service account");
+        cli.deleteAcl(serviceAccount, AclOperation.ALL, AclPermissionType.ALLOW);
+        var aclList = cli.listACLs(serviceAccount);
+        Optional<AclBinding> aclFiltered = CLIUtils.searchAcl(aclList, serviceAccount, AclOperation.ALL, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        assertTrue(aclFiltered.isEmpty());
+    }
+
+    @SneakyThrows
+    @Test(priority = 3)
+    public void testCreateAclGrantAccessForUser() {
+        LOGGER.info("Create grant-access ACL for secondary user");
+        cli.grantAccessAcl(DIFF_USER, TOPIC_NAME, "all");
+        var aclList = cli.listACLs(DIFF_USER);
+        Optional<AclBinding> acl1 = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.READ, AclPermissionType.ALLOW, AclResourceType.GROUP, "*");
+        Optional<AclBinding> acl2 = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.CREATE, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        Optional<AclBinding> acl3 = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.DESCRIBE, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        Optional<AclBinding> acl4 = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.READ, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        Optional<AclBinding> acl5 = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.WRITE, AclPermissionType.ALLOW, AclResourceType.TOPIC, TOPIC_NAME);
+        Optional<AclBinding> acl6 = CLIUtils.searchAcl(aclList, DIFF_USER, AclOperation.DESCRIBE, AclPermissionType.ALLOW, AclResourceType.TRANSACTIONAL_ID, "*");
+        assertTrue(acl1.isPresent());
```

Comment:
maybe also addition of logs to asserts in case of failure to know which one was problem (but there will be one more review once all other problems are resolved.